packages/replay/src/install.ts

```diff
@@ -1,15 +1,12 @@
 // Manage installation of browsers for other NPM packages.
 
 import { spawnSync } from "child_process";
-import dbg from "./debug";
 import fs from "fs";
 import https from "https";
 import path from "path";
-import { BrowserName, Options } from "./types";
+import { BrowserName, Ctx, Options } from "./types";
 import { defer, getDirectory, maybeLog } from "./utils";
 
-const debug = dbg("replay:cli:install");
```

Comment:
In a couple of files here the extended namespace was lost. Those modules are not objects into which I could shove `this._debug`. They export "unrelated" functions/utils. So to avoid the hassle of addressing this right now I decided to avoid some of the namespaces like this as I don't think they were that important. We could bring them back using weakmap-based caching or by refactoring those modules so they gain "instance scope" in which we could keep the extended debug's instance.